physics/lagrange_equipotentials_body.hpp

```diff
@@ -0,0 +1,234 @@
+              /*local_search_tolerance=*/1e-3 * Metre);
+  SpecificEnergy maximum_maximorum = -Infinity<SpecificEnergy>;
+  for (auto const& arg_maximum : arg_maximorum) {
+    auto const maximum = potential(arg_maximum);
+    if (!IsFinite(maximum)) {
+      return absl::OutOfRangeError(absl::StrCat("Improper maximum ",
+                                                DebugString(maximum),
+                                                " at ",
+                                                DebugString(arg_maximum)));
+    }
+    result.maxima.emplace(maximum, arg_maximum);
+    maximum_maximorum = std::max(maximum_maximorum, maximum);
+  }
+
+  BarycentreCalculator<Position<Inertial>, GravitationalParameter>
+      primary_Inertial_position;
+  for (not_null primary : parameters.primaries) {
+    primary_Inertial_position.Add(
+        ephemeris_->trajectory(primary)->EvaluatePosition(t),
+        primary->gravitational_parameter());
+  }
+  BarycentreCalculator<Position<Inertial>, GravitationalParameter>
+      secondary_Inertial_position;
+  for (not_null secondary : parameters.secondaries) {
+    secondary_Inertial_position.Add(
+        ephemeris_->trajectory(secondary)->EvaluatePosition(t),
+        secondary->gravitational_parameter());
+  }
+  Length const r =
+      (secondary_Inertial_position.Get() - primary_Inertial_position.Get())
+          .Norm();
+
+  Position<RotatingPulsating> const primary_position =
+      reference_frame.ToThisFrameAtTimeSimilarly(t).similarity()(
+          primary_Inertial_position.Get());
+  Position<RotatingPulsating> const secondary_position =
+      reference_frame.ToThisFrameAtTimeSimilarly(t).similarity()(
+          secondary_Inertial_position.Get());
+
+  Length primary_radius;
+  if (parameters.primaries.size() == 1) {
+    primary_radius = parameters.primaries.front()->min_radius();
+  } else {
+    for (not_null const primary : parameters.primaries) {
+      primary_radius =
+          std::max(primary_radius,
+                   (primary_Inertial_position.Get() -
+                    ephemeris_->trajectory(primary)->EvaluatePosition(t))
+                           .Norm() +
+                       primary->min_radius());
+    }
+  }
+
+  Length secondary_radius;
+  if (parameters.secondaries.size() == 1) {
+    secondary_radius = parameters.secondaries.front()->min_radius();
+  } else {
+    for (not_null const secondary : parameters.secondaries) {
+      secondary_radius =
+          std::max(secondary_radius,
+                   (secondary_Inertial_position.Get() -
+                    ephemeris_->trajectory(secondary)->EvaluatePosition(t))
+                           .Norm() +
+                       secondary->min_radius());
+    }
+  }
+
+  // TODO(egg): Consider additional wells.
+  std::vector<Equipotential<Inertial, RotatingPulsating>::Well> wells{
+      {secondary_position, secondary_radius / r * (1 * Metre)},
+      {primary_position, primary_radius / r * (1 * Metre)}};
```

Comment:
Move the constants.

Reply:
Not this one: it is not something we want to parameterize, it is instead from the contract of RotatingPulsatingFrame.